### PR TITLE
fully qualify admission resources and kinds

### PR DESCRIPTION
**Pull request by deads2k**
Fully qualifies the `Kind` and `Resource` fields for admission attributes.  The information was getting filtered at the `RESTHandler` before.

@derekwaynecarr

### CODE REVIEW

**Comment by deads2k** [comment]
https://github.com/kubernetes/kubernetes/issues/24528

**Comment by deads2k** [comment]
@k8s-bot unit test this please: https://github.com/kubernetes/kubernetes/issues/24528

File: plugin/pkg/admission/persistentvolume/label/admission_test.go [hunk 1]
```diff
@@ -87,28 +87,28 @@ func TestAdmission(t *testing.T) {
 	}
 
 	// Non-cloud PVs are ignored
-	err := handler.Admit(admission.NewAttributesRecord(&ignoredPV, api.Kind("PersistentVolume"), ignoredPV.Namespace, ignoredPV.Name, api.Resource("persistentvolumes"), "", admission.Create, nil))
+	err := handler.Admit(admission.NewAttributesRecord(&ignoredPV, api.Kind("PersistentVolumes").WithVersion("version"), ignoredPV.Namespace, ignoredPV.Name, api.Resource("persistentvolumes").WithVersion("version"), "", admission.Create, nil))
```
Comment (derekwaynecarr) [inline plugin/pkg/admission/persistentvolume/label/admission_test.go]: why the change to plural throughout here...

Comment (deads2k) [inline plugin/pkg/admission/persistentvolume/label/admission_test.go]: > why the change to plural throughout here...

Typo.  Will sweep them and retag you.

**Comment by deads2k** [comment]
@derekwaynecarr typo fixed.  I didn't see other singular to plurals.

**Comment by deads2k** [comment]
@derekwaynecarr bump

**Comment by derekwaynecarr** [comment]
LGTM, have fun sitting in submit queue.

**Comment by deads2k** [comment]
@k8s-bot unit test this issue #23822
